### ISSUE TITLE
German version of NVDA says "rk" (the braille abbreviation" for a tab in a multi-page dialog or browser context

### ISSUE THREAD

**Issue by MarcoZehe** [nvaccess/nvda#8972]
### Steps to reproduce:

In Firefox:
1. Open a new tab.
2. Press CTRL+L to go to the address bar.
3. Shift+Tab twice.
4. Listen to what NVDA says.

### Actual behavior:

In the German version, NVDA says, as last utterance: "Neuer Tab rk". The last part "rk" is the braille abbreviation for "Registerkarte". "Registerkarte" is what should be spoken.

### Expected behavior:

NVDA says "Registerkarte" instead of "rk".

#### NVDA Installed/portable/running from source:

installed.

#### NVDA version:

NVDA version alpha-16298,c651588e

#### Windows version:

19H1 build 18282.

#### Name and version of other software in use when reproducing the issue:

Firefox, any current version, can also be any other app like Chrome.

#### Other information about your system:

Nothing of relevance.

#### Does the issue still occur after restarting your PC?

Yes.

#### Have you tried any other versions of NVDA?

Previous alphas and 2018.3.2, does not occur, is a recent regression in the German translation, may also be in 2018.4beta.

**Comment by Adriani90** [comment]
@MarcoZehe I have fixed this just two days ago. It will be solved next time when translations will be merged.
Thanks for reporting.

**Comment by leonardder** [comment]
Than, the fix should already be in the beta branch, since Thursday is translation merge day.

**Comment by Adriani90** [comment]
@MarcoZehe can you still reproduce this in NVDA 2018.4 Beta 2? I can confirm that this is not an issue in Beta 2 anymore.

**Comment by leonardder** [comment]
This is fixed in Alpha as well. Closing as works for me.
